mingus/midi/midi_file_in.py

```diff
@@ -100,12 +115,11 @@ def MIDI_to_Composition(self, file):
                         pass
                 elif event["event"] == 9:
                     # note on
-                    n = Note(
-                        notes.int_to_note(event["param1"] % 12),
-                        event["param1"] / 12 - 1,
-                    )
-                    n.channel = event["channel"]
-                    n.velocity = event["param2"]
+                    n = Note(notes.int_to_note(event['param1'] % 12),
+                             np.floor(event['param1'] / 12)) # this was event['param1']/12 - 1 but that gives skewed, incorrect octaves
```

Comment:
I think instead of `floor` we can simply use integer division: `event["param1"] // 12`.

Even if `floor` was needed, the one from NumPy returns a float instead of an int, and pulls in an unnecessary dependency (`numpy` is only required for the FFT module). The `floor` from the built-in `math` module would fit in perfectly.

Reply:
Already fixed on master (due to #64)